src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -146,6 +330,19 @@ static void Fred()
 False
 6
 True
+9
```

Comment:
I understand why the extra spew is now present and that it may be unavoidable. Unfortunately it pushes marginally-understandable test output across the border to inscrutability.

@jcouv has suggested adding comments in the test input source to help a reader correlate the test input and output. (Simply commenting each executable line with the corresponding number in the output would help a lot.) Feel free to create an issue for this and assign it to me.